Python/math/count_of_digits.py

```diff
@@ -0,0 +1,50 @@
+"""
+Python program to get count of digits.
+Count of digits refers to the sum of the frequency of all distinct digits in a number.
+"""
+
+#Function to count digits in the number
+
+def count_of_digits(n):
+    if(n==0):
+      return 1
+
+    #Initializing cnt variable to zero
+
+    cnt = 0
+    while True: 
+        if(n==0):
+          break
+
+        #Increasing the cnt
+
+        cnt+=1 
+
+        #Removing the least significant digit
+
+        n = n//10
+    return cnt
+
+#Taking Input
+
+n = int(input("Enter the number :"))
+
+#Printing Output
+
+print("The count of digits in the given number is: {} ".format(count_of_digits(n)))
```

Comment:
Enclose in a `if __name__ == '__main__'` block

Reply:
done